src/components/Identity/index.js

```diff
@@ -25,15 +35,54 @@ const addIdsContext = (payload, ecid) => {
 const createIdentity = ({ config, logger, cookie }) => {
   // We avoid reading the ECID from the cookie right away, because we
   // need to wait for the user to opt in first.
-  const getEcid = () => cookie.get(ECID_NAMESPACE);
+  const getEcid = () => cookie.get(EXPERIENCE_CLOUD_ID);
   let optIn;
   let responseRequested = false;
   let deferredForEcid;
+  let network;
+  let lifecycle;
+  const customerIDs = {};
+  const getCustomerIDs = () => customerIDs;
+
+  const makeServerCall = event => {
+    const payload = network.createPayload();
+    payload.addEvent(event);
+    const responsePromise = Promise.resolve()
```

Comment:
Can't we just return here instead of the `const`?

Reply:
Done